ui/src/app/metadata/detail/metadata-list/metadata-list.component.ts

```diff
@@ -0,0 +1,193 @@
+
+  title = "Metadata"
+  handleSelectedMetadata?: IJobCode[]|INamedReference[]
+  selectedMetadataType = "category"
+  matchingQuery?: string[]
+
+  typeControl: FormControl = new FormControl(this.selectedMetadataType)
+  columnSort: ApiSortOrder = ApiSortOrder.NameAsc
+
+  from = 0
+  size = 50
+  showSearchEmptyMessage = false
+  resultsLoaded: Observable<PaginatedMetadata> | undefined
+  canDeleteMetadata = this.authService.isEnabledByRoles(ButtonAction.MetadataAdmin)
+
+  searchForm = new FormGroup({
+    search: new FormControl("")
+  })
+  sampleJobCodeResult = new PaginatedMetadata([
+    new ApiJobCode({code: "code1", targetNodeName: "targetNodeName1", frameworkName: "frameworkName1", url: "url1", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code2", targetNodeName: "targetNodeName2", frameworkName: "frameworkName2", url: "url2", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code3", targetNodeName: "targetNodeName3", frameworkName: "frameworkName3", url: "url3", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code4", targetNodeName: "targetNodeName4", frameworkName: "frameworkName4", url: "url4", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code5", targetNodeName: "targetNodeName5", frameworkName: "frameworkName5", url: "url5", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code6", targetNodeName: "targetNodeName6", frameworkName: "frameworkName6", url: "url6", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code7", targetNodeName: "targetNodeName7", frameworkName: "frameworkName7", url: "url7", broad: "broad1", level: "Broad"}),
+    new ApiJobCode({code: "code8", targetNodeName: "targetNodeName8", frameworkName: "frameworkName8", url: "url8", broad: "broad1", level: "Broad"}),
+  ], 8)
+
+  sampleNamedReferenceResult = new PaginatedMetadata([
+    new ApiNamedReference({id: "id1", framework: "framework1", name: "name1", type: MetadataType.Category, value: "value1"}),
+    new ApiNamedReference({id: "id2", framework: "framework2", name: "name2", type: MetadataType.Category, value: "value2"}),
+    new ApiNamedReference({id: "id3", framework: "framework3", name: "name3", type: MetadataType.Category, value: "value3"}),
+    new ApiNamedReference({id: "id4", framework: "framework4", name: "name4", type: MetadataType.Category, value: "value4"}),
+    new ApiNamedReference({id: "id5", framework: "framework5", name: "name5", type: MetadataType.Category, value: "value5"}),
+    new ApiNamedReference({id: "id6", framework: "framework6", name: "name6", type: MetadataType.Category, value: "value6"}),
+    new ApiNamedReference({id: "id7", framework: "framework7", name: "name7", type: MetadataType.Category, value: "value7"}),
+    new ApiNamedReference({id: "id8", framework: "framework8", name: "name8", type: MetadataType.Category, value: "value8"}),
+  ], 8)
+
+  results: PaginatedMetadata = this.sampleJobCodeResult
```

Comment:
The default value in select is a category (named reference), so this line should be results: PaginatedMetadata = this.sampleNamedReferenceResult

Reply:
nice catch, yeah I forgot to check on the default selection.